packages/terra-section-header/src/SectionHeader.module.scss

```
@@ -152,3 +153,19 @@
     }
   }
 }
+
+.toggle-button {
+  background: none;
+  border: 0;
+  color: inherit;
+  cursor: pointer;
+  padding: 0;
+  text-align: left;
+}
+
+.fit-block {
+  .icon-wrapper {
+    display: inline-flex;
+    font-size: 8px;
```

```suggestion
    font-size: 8px;
```
do we need font-size for icon ..??

---

Earlier i got some minor font difference on the icon but now it looks good. So I have removed this font size.